packages/react/src/components/InAppMessaging/CloseIconButton/types.ts

```diff
@@ -0,0 +1,8 @@
+import { LinkProps } from '../../../primitives';
```

Comment:
See note about preferring Button primitive.

Reply:
@cshfang Should this be: 
```suggestion
import { ButtonProps } from '../../../primitives';
```
now that we're using a Button?